src/core/backend/pearl/settings.rs

```diff
@@ -12,12 +12,16 @@ pub(crate) struct Settings {
 impl Settings {
     pub(crate) fn new(config: &NodeConfig, mapper: Arc<Virtual>) -> Self {
         let config = config.pearl().clone();
-        let disk_path = mapper
-            .get_disk(&config.alien_disk())
-            .expect("cannot find alien disk in config")
-            .path();
-        let alien_folder =
-            format!("{}/{}/", disk_path, config.settings().alien_root_dir_name()).into();
+        let alien_disk = config.alien_disk();
+        let alien_folder = if alien_disk.is_empty() {
```

Comment:
If I remember it right, alien_disk won't be empty, if will be `~`. And lets wrap into Option<String>, because it is an optional parameter now.

Reply:
I set method for default alien_disk, which returns empty string, so it will be empty, but, you are right, that solution with option is more elegant, I'll fix that